straxen/url_config.py

```diff
@@ -264,14 +269,16 @@ def evaluate_dry(cls, url: str, **kwargs):
         :keyword: any additional kwargs are passed to self.dispatch (see example)
         :return: evaluated value of the URL.
         """
-        if cls.PLUGIN_ATTR_PREFIX in url:
-            raise ValueError(
-                'You specified at least one parameter that depends on the '
-                'plugin. We cannot fetch this in the dry run. Try replacing '
-                'e.g. plugin.run_id=027000 with run_id=027000 (or pass as an '
-                'extra kwargs)')
         url_arg, url_kwarg = cls.split_url_kwargs(url)
-        return cls.dispatch(cls, url_arg, **url_kwarg, **kwargs)
+
+        combined_kwargs = dict(url_kwarg, **kwargs)
+        for k,v in combined_kwargs.items():
+            if isinstance(v, str) and cls.PLUGIN_ATTR_PREFIX in v:
+                raise ValueError(f'The URL parameter {k} depends on the plugin'
+                                'You must specify the value for this parameter'
+                                'for this URL to be evaluated correctly.')
```

Comment:
perhaps worth keeping the example? At least for some people might be lost in abstraction here?

Reply:
Sure, not sure why i removed it. Maybe i was too lazy to reformulate it with kwarg substitution allowed...